libraries/helpers.rb

```diff
@@ -20,6 +20,13 @@ def fs_type(mount)
     def fs_remote?(mount)
       fs_type(mount) == 'nfs' ? true : false
     end
+
+    # Validates passed id.
+    #
+    # @return [Numeric, String]
+    def validate_id(id)
+      id.to_i.to_s == id ? id.to_i : id
```

Comment:
What's the input scenario where we can't convert to an int? I'm trying to understand how we can be very defensive here and fail in a way that lets the user know they're providing bad input data.